README.md

```diff
@@ -1,2 +1,36 @@
+
+3. Launch "Eclipse IDE for RCP and RAP Developers" and create a workspace. (Wherever you like. It seems appropriate to create the workspace in the openEQUELLA-reporting-plugin directory. Note that the workspace .metadata directory is already included in the .gitignore file)
+
+4. In the Project Explorer window, select "Import projects..." -> "General" -> "Existing Projects into Workspace".  Browse to the root of openEQUELLA-reporting-plugin repository and select that folder.  Make sure all the discovered projects are selected and then click Finish.
+
+5. At this point, you will be seeing compilation errors (missing imports), so we need to add a "Target Platform".  Go to Window->Preferences->Plug-in Development->Target Platform.  You will note that it currently says "Running Platform (Active)".  This means you are using the current development Eclipse's plugins as dependencies, which doesn't include the BIRT plugins.  
+
+6. From the screen in step 5, click the "Add..." button, choose "Nothing" as the base, and click "Next".  Name the Target Platform "BIRT" (or "Mr. T", whatever, it doesn't matter) and click "Add..." on the Locations tab.
+
+7. From the "Add Content" dialog that comes up, choose "Installation" and browse to the root of the BIRT folder (the folder that includes the eclipse.ini and executable).  Click "Finish", but remain in the "New Target Definition" dialog.  
+
+8. If we were to finish creating the Target Platform now, our plugins would mostly build, but would still be missing XStream dependencies.  From the "New Target Definition" dialog, click "Add..." again, but this time choose "Software Site".  Click "Add..." and an "Add Repository" dialog comes up.  Enter "Orbit"* in the Name field, and paste https://download.eclipse.org/tools/orbit/downloads/drops/R20200224183213/repository into the Location field.  Once you click "Add", wait for the list of plugins to download and display.
+
+9. In the "type filter text" box, enter "xstream", and tick the box next to XStream that displays in the tree below.  Now click "Finish".
+
+10. From the "Edit Target Definition" dialog, click "Finish" again.  Now click the checkbox against your newly added Target definition.  Click "Apply and Close".  After a short build time, your compilation errors should disappear.
+
+11. Right click on the "openEQUELLA Reporting Plugins" project and choose "Export..."->"Plug-in Development"->"Deployable features". Select a destination directory and click Finish.
+
+12. The contents of the "plugins" sub-directory of the export directory in the previous step are deployable to BIRT.  Copy all of these jar files into the "dropins" directory of you BIRT designer installation.
+
+13. Start the BIRT designer and follow the section at the top of this document titled "Report Designers".
```

Comment:
The links in the 'Report Designers' section are old and the first is broken. Might as well update as part of this.

The first should be: https://openequella.github.io/

And ideally rather than pointing into the repo, might as well match the first and go to the actual doco site: https://openequella.github.io/tutorials/reporting/index.html